dove/src/cmd/build.rs

```diff
@@ -97,25 +100,30 @@ impl Cmd for Build {
 
         // Build move files...
         let sender = ctx.account_address()?;
-        let Artifacts { files, prog } = MoveBuilder::new(
+        let Artifacts { files, env, prog } = MoveBuilder::new(
             ctx.dialect.as_ref(),
             Some(sender),
             StaticResolver::new(dep_list),
         )
-        .build(&source_ref);
+        .build(&source_ref, self.doc);
 
         match prog {
             Err(errors) => {
                 let mut writer = StandardStream::stderr(ColorChoice::Auto);
                 output_errors(&mut writer, files, errors);
-                Err(anyhow!("could not compile:{}", ctx.project_name()))
+                return Err(anyhow!("could not compile:{}", ctx.project_name()));
             }
             Ok(compiled_units) => {
                 // Verify and store compilation results...
                 self.verify_and_store(&ctx, files, compiled_units, &exclude_modules)?;
-                Ok(())
             }
         }
+
+        if let Some(env) = env {
```

Comment:
Is there any reason we need **not** to create GlobalEnv?

Reply:
GlobalEnv is used only for generating documentation and is not compatible with executor.

Reply:
Okay, I'm just concerned about increasing code complexity because of additional if branches everywhere.

Reply:
This is a temporary version. We are changing the test system. After that, these conditions will be deleted.